.gitignore

```diff
@@ -8,6 +8,5 @@ Cargo.lock
 !.vscode/launch.json
 !.vscode/extensions.json
 
-/public
```

Comment:
We want the public directory to be ignored. When the action builds, the site the public directory gets committed to the gh-pages branch

Reply:
because dioxus-cli default use `public` dir to put some file you want to transfer to gh-pages, so you mean we change a dir name ?